README.md

```diff
@@ -154,17 +154,11 @@ OPTIONS:
             [default: 1.1]
         --economic-viability-strategy <economic-viability-strategy>
             How to calculate the economic viability constraints. `Dynamic` means that current native token price is
-            taken into account while `Static` means that fallback_min_avg_fee_per_order and fallback_max_gas_price will
+            taken into account while `Static` means that static_min_avg_fee_per_order and static_max_gas_price will
```

Comment:
This was a correct change, but somehow got reverted back to fallback in a later PR.